src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -211,6 +216,15 @@ private void validateSchema(final EventTypeBase eventType) throws InvalidEventTy
         }
     }
 
+    @VisibleForTesting
```

Comment:
@v-stepanov I usually do it the way you described, but the reason was that JsonParser returns a pretty good described error and points to the position where the error happened. my thoughts were it is better to have this annotation and keep the InvalidEventTypeException out of the separate class. I have thought it over again and I think if I name the method properly it will be fine to have the exception in separate class.
Here is a fix 57fa20f